### PR TITLE
🐛 main.go: improve logging for configuration of global auth

### PR DESCRIPTION
**Pull request by joelanford**
# Description

## Reviewer Checklist

- [ ] API Go Documentation
- [ ] Tests: Unit Tests (and E2E Tests, if appropriate)
- [ ] Comprehensive Commit Messages
- [ ] Links to related GitHub Issue(s)

### CODE REVIEW

**Review by tmshort — APPROVED** [review]
/lgtm

**Review by everettraven — APPROVED** [review]

File: cmd/manager/main.go [hunk 1]
```diff
@@ -319,11 +319,13 @@ func (f finalizerFunc) Finalize(ctx context.Context, obj client.Object) (crfinal
 func authFilePathIfPresent(logger logr.Logger) string {
 	_, err := os.Stat(authFilePath)
 	if os.IsNotExist(err) {
+		logger.Info("auth file not found, skipping configuration of global auth file", "path", authFilePath)
 		return ""
 	}
 	if err != nil {
-		logger.Error(err, "could not stat auth file path", "path", authFilePath)
+		logger.Error(err, "unable to stat auth file path", "path", authFilePath)
```
Comment (LalatenduMohanty) [inline cmd/manager/main.go:326]: `unable to stat auth file path` is kind of meaning less in a way. How would the user know that we are talking about os.stat and what does that mean.

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

**Review by LalatenduMohanty — APPROVED** [review]
/lgtm

**Pull request merged**